tests/unit/test_entity_extractor.py

```
@@ -1,10 +1,12 @@
 import sys
+from pathlib import Path
 
-sys.path.append("/home/cbwinslow/CascadeProjects/opendiscourse")
+# Ensure project root is on sys.path
+sys.path.append(str(Path(__file__).resolve().parents[2]))
 
 import logging
```

Flake8 (7.2.0)

[error] module level import not at top of file

(E402)